### PR TITLE
[RHCLOUD-27608] apply endpoint disabling logic to http connectors

### PR DESCRIPTION
**Pull request by g-duval**

### CODE REVIEW

**Review by MikelAlejoBR — APPROVED** [review]

**Review by gwenneg — CHANGES_REQUESTED** [review]
I need to check something before this is merged.

File: connector-common-http/src/main/java/com/redhat/cloud/notifications/connector/http/HttpOutgoingCloudEventBuilder.java [hunk 1]
```diff
@@ -27,7 +26,7 @@ public void process(Exchange exchange) throws Exception {
             Message in = exchange.getIn();
             JsonObject cloudEvent = new JsonObject(in.getBody(String.class));
             JsonObject data = new JsonObject(cloudEvent.getString("data"));
-            data.getJsonObject("details").put(STATUS_CODE, exchange.getProperty(HTTP_STATUS_CODE));
+            data.getJsonObject("details").put(HTTP_STATUS_CODE, exchange.getProperty(HTTP_STATUS_CODE));
```
[the remark this answers is not in the document]
Comment (g-duval) [reply connector-common-http/src/main/java/com/redhat/cloud/notifications/connector/http/HttpOutgoingCloudEventBuilder.java:29]: Thanks, I updated it on engine as well

File: .rhcicd/clowdapp-connector-splunk.yaml [hunk 2]
```diff
@@ -81,6 +81,8 @@ objects:
           value: ${NOTIFICATIONS_CONNECTOR_SEDA_CONCURRENT_CONSUMERS}
         - name: NOTIFICATIONS_CONNECTOR_SEDA_QUEUE_SIZE
           value: ${NOTIFICATIONS_CONNECTOR_SEDA_QUEUE_SIZE}
+        - name: NOTIFICATIONS_CONNECTOR_HTTP_DISABLE_FAULTY_ENDPOINTS
```
Comment (gwenneg) [inline .rhcicd/clowdapp-connector-splunk.yaml:84]: The SNow template will need the same thing.

Comment (g-duval) [reply .rhcicd/clowdapp-connector-splunk.yaml:84]: Sure, sorry I forgot it

**Review by gwenneg — APPROVED** [review]
LGTM

**Pull request merged**